moban/engine_handlebars.py

```diff
@@ -0,0 +1,112 @@
+import os
+
+import moban.utils as utils
+import moban.reporter as reporter
+import moban.constants as constants
+from lml.loader import scan_plugins_regex
+from lml.plugin import PluginInfo
+from moban.engine_factory import (
+    MOBAN_ALL,
+    BUILTIN_EXENSIONS,
+    Context,
+    Strategy,
+    expand_template_directory,
+    expand_template_directories,
+    verify_the_existence_of_directories
+)
+
+from pybars import Compiler
+
+
+@PluginInfo(
+    constants.TEMPLATE_ENGINE_EXTENSION, tags=["handlebars", "hbs"]
+)
+class EngineHandlebars(object):
```

Comment:
As you see, this class has duplicated the code. With DIY principle, please find a way to re-use code instead of copy-n-paste code. Code reuse is one of software engineering principles.